apps/sms/services/README.md

```diff
@@ -0,0 +1,91 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+There are 3 layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Services define high-level semantics to access the model. They are the only
```

Comment:
Services (as in Service Layer) are already "real" services (as in Bridge-managed services -- we need a better terminology :p) ,and will use bridge.service. They may use bridge.client to call other services/managers/wrappers.

What I'm not sure yet is whether the "Manager" layer should be bridge-managed services or merely JS libs that we can import into Service layer services. I'd rather take the simpler road for a start (so "Managers" are reusable libs). Still the distinction is important to me, so that we don't duplicate code and think in reusable components rather than organically refactor things because technically it's better.

Tell me if it's clearer :)